### PR TITLE
Added post-memories-app

### LINKED ISSUE #413: Title: Post Memories App
### Is your feature request related to a problem? Please describe.

I want to make a web app which lets people form anywhere post their memories/experiences on a single platform where everyone can see each others posts.

### Describe the solution you'd like.

I am using the following tech stack:
1) React
2) Node.js
3) Express.js
4) MongoDB

### Describe alternatives you've considered.

Null

### Add any other context or screenshots about the feature request here.

Hi @Kushal997-das, I am an active contributor in GSSOC'22. So please assign this issue to me, thank you.

### PR DESCRIPTION
**Pull request by rahulNetkar**
### Have you read the [Contributing Guidelines](https://github.com/Kushal997-das/Project-Guidance/blob/main/CONTRIBUTING.md) ?

Yes, I have read the guidelines

### Description

A web app which lets people from anywhere post their memories/experiences on a single platform where everyone can see each others posts.

Fixes #413

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

None

### CODE REVIEW

File: Web Development/Intermediate/post-memories-app/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+![1](https://badgen.net/badge/Programming-Language/javascrip/blue)
+![5](https://badgen.net/badge/Application/Post-Memories/orange)
+![2](https://badgen.net/badge/Frontend/React/red)
+![3](https://badgen.net/badge/Backend/Node.js/yellow)
+![4](https://badgen.net/badge/Database/MongoDB/pink)
+![6](https://badgen.net/badge/Status/complete/green)
+
+# ❣ Post Memories App ❣
+
```
Comment (Kushal997-das) [inline Web Development/Intermediate/post-memories-app/README.md:9]: Add a brief description about about your project here !

File: Web Development/Intermediate/post-memories-app/README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+![1](https://badgen.net/badge/Programming-Language/javascrip/blue)
```
Comment (Kushal997-das) [inline Web Development/Intermediate/post-memories-app/README.md:1]: Spelling error `javascript`

**Comment by rahulNetkar** [comment]
@Kushal997-das Added description and corrected typo, please check it.

**Review by Kushal997-das — APPROVED** [review]
@photon149 have a look !

**Pull request merged**